test/test_wiki.rb

```diff
@@ -303,18 +303,41 @@
     assert @wiki.page("Saruman")
   end
 
-  test "write page is not allowed to overwrite file" do
+  test "write page is not allowed to overwrite pages by default" do
     @wiki.write_page("Abc-Def", :markdown, "# Gollum", commit_details)
     assert_raises Gollum::DuplicatePageError do
       @wiki.write_page("Abc-Def", :markdown, "# Gollum", commit_details)
     end
+    @wiki.write_page("subdir/Abc-Def", :markdown, "# Gollum", commit_details)
+    assert_raises Gollum::DuplicatePageError do
+      @wiki.write_page("subdir/Abc-Def", :markdown, "# Gollum", commit_details)
+    end
     assert_nothing_raised Gollum::DuplicatePageError do
       @wiki.write_page("Abc-Def", :textile, "# Gollum", commit_details)
     end
     assert_nothing_raised Gollum::DuplicatePageError do
       @wiki.write_page("abc-def", :markdown, "# Gollum", commit_details)
     end
   end
+  
+  test "write file is not allowed to overwrite files by default" do
+    @wiki.write_file("Abc-Def.file", "# Gollum", commit_details)
+    assert_raises Gollum::DuplicatePageError do
+      @wiki.write_file("Abc-Def.file", "# Gollum", commit_details)
+    end
+    @wiki.write_file("subdir/Abc-Def.file", "# Gollum", commit_details)
+    assert_raises Gollum::DuplicatePageError do
+      @wiki.write_file("subdir/Abc-Def.file", "# Gollum", commit_details)
+    end
+  end
+
+  test "write page can overwrite pages if overwrite option is enabled" do
+    omit('Not yet implemented.')
```

Comment:
Ok to omit this for now, but for my comprehension: is the plant to implement this in a later release, or a separate PR?

Reply:
In a separate PR, and likely in a later release as well. Will actually remove these tests from this PR.